### ISSUE TITLE
I can not open project

### ISSUE THREAD

**Issue by xuepeng9** [spyder-ide/spyder#17976]
### What steps will reproduce the problem?

the window was occured when i click to open project

### Traceback
```python-traceback
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\lib\site-packages\spyder\plugins\projects\plugin.py", line 276, in <lambda>
    triggered=lambda v: self.open_project())
  File "C:\ProgramData\Anaconda3\lib\site-packages\spyder\plugins\projects\plugin.py", line 433, in open_project
    project_type_class = self._load_project_type_class(path)
  File "C:\ProgramData\Anaconda3\lib\site-packages\spyder\plugins\projects\plugin.py", line 864, in _load_project_type_class
    config.read(fpath)
  File "C:\ProgramData\Anaconda3\lib\configparser.py", line 697, in read
    self._read(fp, filename)
  File "C:\ProgramData\Anaconda3\lib\configparser.py", line 1020, in _read
    for lineno, line in enumerate(fp, start=1):
UnicodeDecodeError: 'gbk' codec can't decode byte 0xb0 in position 198: illegal multibyte sequence
```

## Versions

* Spyder version: 5.1.5 
* Python version: 3.9.12
* Qt version: 5.9.7
* PyQt5 version: 5.9.2
* Operating System: Windows 10

### Dependencies

```

# Mandatory:
atomicwrites >=1.2.0          :  1.4.0 (OK)
chardet >=2.0.0               :  4.0.0 (OK)
cloudpickle >=0.5.0           :  2.0.0 (OK)
cookiecutter >=1.6.0          :  1.7.3 (OK)
diff_match_patch >=20181111   :  20200713 (OK)
intervaltree >=3.0.2          :  3.1.0 (OK)
IPython >=7.6.0               :  8.2.0 (OK)
jedi >=0.17.2;<0.19.0         :  0.18.1 (OK)
jsonschema >=3.2.0            :  4.4.0 (OK)
keyring >=17.0.0              :  23.4.0 (OK)
nbconvert >=4.0               :  6.4.4 (OK)
numpydoc >=0.6.0              :  1.2 (OK)
paramiko >=2.4.0              :  2.8.1 (OK)
parso >=0.7.0;<0.9.0          :  0.8.3 (OK)
pexpect >=4.4.0               :  4.8.0 (OK)
pickleshare >=0.4             :  0.7.5 (OK)
psutil >=5.3                  :  5.8.0 (OK)
pygments >=2.0                :  2.11.2 (OK)
pylint >=2.5.0;<2.10.0        :  2.9.6 (OK)
pyls_spyder >=0.4.0           :  0.4.0 (OK)
pylsp >=1.2.2;<1.3.0          :  1.2.4 (OK)
pylsp_black >=1.0.0           :  None (OK)
qdarkstyle =3.0.2             :  3.0.2 (OK)
qstylizer >=0.1.10            :  0.1.10 (OK)
qtawesome >=1.0.2             :  1.0.3 (OK)
qtconsole >=5.1.0             :  5.3.0 (OK)
qtpy >=1.5.0                  :  2.0.1 (OK)
rtree >=0.9.7                 :  0.9.7 (OK)
setuptools >=49.6.0           :  61.2.0 (OK)
sphinx >=0.6.6                :  4.4.0 (OK)
spyder_kernels >=2.1.1;<2.2.0 :  2.1.3 (OK)
textdistance >=4.2.0          :  4.2.1 (OK)
three_merge >=0.1.1           :  0.1.1 (OK)
watchdog >=0.10.3             :  2.1.6 (OK)
zmq >=17                      :  22.3.0 (OK)

# Optional:
cython >=0.21                 :  0.29.28 (OK)
matplotlib >=2.0.0            :  3.5.1 (OK)
numpy >=1.7                   :  1.21.5 (OK)
pandas >=1.1.1                :  1.4.2 (OK)
scipy >=0.17.0                :  1.7.3 (OK)
sympy >=0.7.3                 :  1.10.1 (OK)
```

**Comment by xuepeng9** [comment]
UnsatisfiableError: The following specifications were found
to be incompatible with the existing python installation in your environment:

Specifications:

  - spyder=4.1.5 -> python[version='>=3.6,<3.7.0a0|>=3.8,<3.9.0a0|>=3.7,<3.8.0a0']

Your python: python=3.9

**Comment by xuepeng9** [comment]
> Hey @xuepeng9, this was detected incorrectly by our bot but it's fixed in our latest version (5.3.0).
> 
> Since it's still not available in Anaconda, you have two options:
> 
> 1. Use our Windows installer, which you can find [here](https://github.com/spyder-ide/spyder/releases).
> 2. Create a new environment with conda-forge packages. For that, please close Spyder, open the Anaconda Prompt and run the following commands there:
>    ```
>     conda create -n spyder-cf -c conda-forge spyder
>     conda activate spyder-cf
>     spyder
>    ```

Thanks for your suggestion,the problem had been sloved,but a new problem that I can't call any uninstalled third-party libraries,althought these are installed in anaconda.so I want to know when the latest version(5.3.0) will be updated in anaconda?

**Comment by ccordoba12** [comment]
> so I want to know when the latest version(5.3.0) will be updated in anaconda?

We have no idea about that, sorry.
